photo-gallery/src/App.js

```diff
@@ -21,20 +21,21 @@ render () {
     };
 
     return (
-        
+        <>
             <Header pan={pan}/>
-        //    {/* <main>
-        //        <section className="options">
-        //            <select className="beast-type" onChange={handleChange}/>
-        //            <option value="" defaultValue>
-        //                All Beast
-        //            </option>
-        //        </section>
-        //        <section className="list-section">
-        //            <ul className="hornedBeast">{hornedBeastNodes</ul>
-        //        </section>
-        //    </main> */}
-        
-    )
-}
-}
\ No newline at end of file
+           
+       
+                <main>
+               <section className="options">
+                   <select className="beast-type" onChange={handleChange}/>
+                   <option value="" defaultValue>
+                       All Beast
+                   </option>
+               </section>
+               <section className="list-section">
```

Comment:
This section should actually be it's own component — ImageList. 
`<section className="list-section">  
       <ul className="hornedBeast">{hornedBeast}> </ul>
  </section>`